lib/all-badge-examples.js

```diff
@@ -109,6 +109,16 @@ const allBadgeExamples = [
       name: 'Build'
     },
     examples: [
+      {
+        title: 'Distelli build',
+        previewUri: '/distelli/build/ORG/APPLICATION-NAME.svg?token=TOKEN',
+        exampleUri: '/distelli/build/ORG/APPLICATION-NAME.svg?token=TOKEN'
```

Comment:
The preview URI will render as a badge, so it would need a real org, app name, and token.

Reply:
I only have a dev subscription on Distelli, so I can not add more users. Not sure if the API support read only tokens, I will ask Distelli about this.

Reply:
Distelli only have tokens with full permissions.

Reply:
Hmm. Using full-access token is unfortunate. Of course anyone looking at the badge would see the token, plus sending it to the Shields server is a security risk.

Do you think a status-only token is something they might add?

Reply:
I have asked them about this, but not received any feedback on this matter. I will update when I know more. They told me that status badges was on their roadmap.

Reply:
They have added RO tokens to their road map. ETA minimum 3 weeks, they will notify me.

Reply:
That is great news!

Reply:
How about I close this, and you can reopen when you hear from them?